File: Code/cpp/power.cpp
#include<iostream>
#include<cmath>
using namespace std;
int main()
{
    int base, exponent;

    cout << "Enter Base: ";
    cin >> base;
    cout << "Enter Exponent: ";
    cin >> exponent;

    cout << pow(base, exponent);
    return 0;
}

File: Code/java/power.java
import java.util.Scanner;

class Power
{
    public static void main(String[] args)
    {
        int base, exponent;
        Scanner scanner = new Scanner(System.in);

        System.out.println("Enter Base: ");
        base = scanner.nextInt();
        System.out.println("Enter Exponent: ");
        exponent = scanner.nextInt();

        System.out.println(Math.pow(base, exponent));

        scanner.close();
    }
}